comeet/UI/RoomsList/RoomsListViewController.swift

```diff
@@ -148,7 +139,19 @@ private extension RoomsListViewController {
         title = viewModel.title()
         
         viewModel.reloadBinding = { [weak self] in
-            self?.tableView.reloadSections([0], with: UITableViewRowAnimation.fade)
+            
+            if viewModel.roomsCount() > 0 {
+                let userDefault = UserDefaults.standard
+                if let weakSelf = self {
+                    weakSelf.guideView.isHidden = false
+                    let isGuideShown = userDefault.bool(forKey: "isRoomsGuideShown")
```

Comment:
Please push the state manipulation into the viewmodel.
Also, I don't see any code that makes tapping on the guide screen dismiss it.